docs/source/en/perf_infer_gpu_multi.md

```diff
@@ -37,9 +43,257 @@ rendered properly in your Markdown viewer.
 
 </details>
 
-Set `tp_plan="auto"` in [`~AutoModel.from_pretrained`] to enable tensor parallelism for inference.
+## Tensor parallelism in-depth
+
+Our implementation of tensor parallelism is built on top of `torch.distributed` package. We heavily utilize abstractions as: `DeviceMesh` or `DTensor` to provide a simple and extensible interface to the user.
```

Comment:
We should say that it is agnostic, but the onces we implemented rely on torch distributed